plugins/no-content/no-content.js

```diff
@@ -0,0 +1,36 @@
+const stylelint = require('stylelint');
+
+const ruleName = 'plugin/no-content';
+
+const messages = stylelint.utils.ruleMessages(ruleName, {
+  rejected: 'You must not use the `content` property to hard-code unlocalized text',
+});
+
+module.exports = stylelint.createPlugin(ruleName, (primaryOption) => {
+  return function(root, result) {
+    const validOptions = stylelint.utils.validateOptions(result, ruleName, {
+      actual: primaryOption,
+    });
+
+    if (!validOptions) {
+      return;
+    }
+
+    const allowedContentValues = [
+      "''",
+      '""',
+      'none',
```

Comment:
We want to forbid usage of:

```
/* <string> value */
content: "prefix";
```

But do we really want to forbid usage of _all_ of these?

```
/* Keywords that cannot be combined with other values */
content: normal;
content: none;

/* <url> value */
content: url("http://www.example.com/test.png");

/* <counter> values */
content: counter(chapter_counter);
content: counters(section_counter, ".");

/* attr() value linked to the HTML attribute value */
content: attr(value string);

/* Language- and position-dependent keywords */
content: open-quote;
content: close-quote;
content: no-open-quote;
content: no-close-quote;

/* Except for normal and none, several values
   can be used simultaneously */
content: open-quote chapter_counter;

/* Global values */
content: inherit;
content: initial;
content: unset;

Reply:
yeah, we should allow those. The only thing we want to block is translatable strings. So making a regex to look for non empty strings should be all we want to block.

Reply:
Yeah good point. We shouldn't allow `*-quote` though because Firefox and Edge don't localize quotes properly.

Reply:
Nice catch